### PR TITLE
First Pass at Table Graph Styles

### PR DESCRIPTION
**Pull request by alexpaxton**
- [ ] Rebased/Mergable
  - [ ] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### The Problem
- Table graph controls need styles
- Visualization type selector needs SVG for table graphs

### The Solution
screen shot 2018-03-05 at 1 02 38 pm

### CODE REVIEW

File: ui/src/dashboards/components/GraphOptionsCustomizableColumn.js [hunk 1]
```diff
@@ -8,15 +8,16 @@ const GraphOptionsCustomizableColumn = ({
...
+      <div className="column-controls--label">
         {originalColumnName}
       </div>
       <InputClickToEdit
         value={newColumnName}
-        wrapperClass="fancytable--td orgs-table--name"
+        wrapperClass="column-controls-input"
         onUpdate={onColumnRename}
         placeholder="Rename..."
+        appearAsNormalInput={true}
```
Comment (ebb-tide) [inline ui/src/dashboards/components/GraphOptionsCustomizableColumn.js]: hahahah. I feel like we might all have an `appearAsNormal` property that we can turn on and off.

File: ui/src/dashboards/components/GraphOptionsThresholds.js [hunk 2]
```diff
@@ -19,8 +19,8 @@ const GraphOptionsThresholds = ({
   onDeleteThreshold,
 }) => {
   return (
-    <div>
-      <label>Thresholds</label>
+    <div className="gauge-controls graph-options-group">
```
Comment (ebb-tide) [inline ui/src/dashboards/components/GraphOptionsThresholds.js]: Is there any way to get references to gauge controls out of graphtable stuff? Or--- could the solution be to name things more generically so it fits both in table graph and gauge controls? We could also take another pass at this when these components are complete. (this work style is an experiment)

Comment (alexpaxton) [reply ui/src/dashboards/components/GraphOptionsThresholds.js]: Yeah I think there's a more generic reusable naming scheme that works for single-stat / gauge / table. I'm most likely going to tackle that closer to the end

File: ui/src/dashboards/components/GraphOptionsCustomizableColumn.js (hunk 1 above)
Comment (alexpaxton) [reply ui/src/dashboards/components/GraphOptionsCustomizableColumn.js]: 😺

**Pull request merged**